data/saas/config/stripe_config.yml

```diff
@@ -37,6 +37,28 @@ saas_config:
           - name: email
             identity: email
         data_path: data
+      update:
+        method: POST
+        path: /v1/customers/<customer_id>
+        headers:
+          - name: Content-Type
+            value: application/x-www-form-urlencoded
+        param_values:
+          - name: customer_id
+            references:
+              - dataset: stripe_connector_example
+                field: customer.id
+                direction: from
+        body: '{<all_object_fields>}'
+      delete:
+        method: DELETE
+        path: /v1/customers/<customer_id>
+        param_values:
+          - name: customer_id
+            references:
+              - dataset: stripe_connector_example
+                field: customer.id
+                direction: from
```

Comment:
Asked elsewhere, but why are we supporting both update and delete simultaneously? Which one should someone use for masking?

Reply:
We can use either the update or delete, based on the value of `config.execution.MASKING_STRICT`.

Reply:
Can you clarify that both do not have to be specified for masking to work? I don't think this is clear in the docs, that only one needs to be defined, and that only one will be used.

Reply:
Sorry! Your original question just clicked for me. I removed the deletes from the endpoints that already have an update.

Reply:
okay great, thanks, I think this will help clear up confusion for people looking at the stripe yaml!